index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/71e053ea26.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+<body>
```

Comment:
[OPTIONAL] Please remove the default 8px margin around the body, it creates overflowing. ↓
![Screen Recording 2023-01-12 at 9 31 20 PM](https://user-images.githubusercontent.com/25887253/212155197-1a16dd1b-0aad-4e35-903f-2f4b89da4ec3.gif)